app/decorators/staff/program_session_decorator.rb

```diff
@@ -56,7 +56,7 @@ def abstract_markdown
   def scheduled_for
```

Comment:
One observation about this is that it add commas when info is missing as well as attempts to convert several objects to strings when that might not make sense.  Likely all minor issues but when I build up a string this way I like to use this approach:
```
parts = []
if object.time_slot
  ts = object.time_slot
  parts << "Day #{ts.conference_day}" if ts.conference_day
  parts << ts.start_time.to_s(:time) if ts.start_time
  parts << ts.room.name if ts.room
end
parts.join(", ")  
```